src/main/webapp/components/NoteGridCollection.vue

```diff
@@ -18,11 +18,31 @@ module.exports = {
       type: Array,
       required: true,
     },
+    filters: Array,
+  },
+  computed: {
+    activeNotes: function() {
+      // Filter the notes that get shown based on filter provided by parent.
+      if (this.filters && this.filters.length) {
+        return this.notes.filter(this.noteFilter);
+      } else {
+        return this.notes;
+      }
```

Comment:
nit: Don't need `else` if you're returning from inside the `if`.
```suggestion
      if (this.filters && this.filters.length) {
        return this.notes.filter(this.noteFilter);
      }
      return this.notes;
```

Reply:
Thanks for the suggestion :)